### DISCUSSION TITLE
Whitespace control with triple {

### DISCUSSION THREAD

**Discussion by MarkGTI** [rfennell/AzurePipelines#1276] [Q&A]
Hi All

I need to get these two expressions on a single line:

**{{lookup this.fields 'Custom.ReleaseNotesTitle'~}}:**{{~{lookup this.fields 'Custom.ReleaseNotes'}~}}

Result should be **title:** text descriptiuons

the second expression always starts a new line

any ideas?

thanks

mark

**Comment by rfennell** [comment]
My guess is there is a`\n` somewhere. You could use one of the [Handbar helpers](https://github.com/helpers/handlebars-helpers) that I ship in the task to do a find and replace. Or write your own extension  to do the processing you require
